random/random_test.go

```diff
@@ -1,26 +1,25 @@
-package random_test
```

Comment:
The idea behind `_test` packages is to write such test that rely on public API of the package being tested. This way we reduce the temptation e.g. to change unexported fields of a struct. Having tests that rely on accessing unexported fields or calling unexported methods makes it harder later on with refactoring, since changing internal API is harder due to this dependency on tests.

Reply:
I couldn't get the test to build if the test was in its separate package.
When I tried:
```
go test github.com/scylladb/scylla-bench/random
```
It would fail at the line where the test imports the random package, saying that package cannot be found.
Admittedly I am a go n00b so perhaps I was missing something obvious.

Reply:
Maybe GOPATH was misconfigured when you tried first time? It should work.

```bash
$ go test github.com/scylladb/scylla-bench/random
ok  	github.com/scylladb/scylla-bench/random	0.005s
$ pwd
/Users/rjeczalik/src/github.com/scylladb/scylla-bench
$ echo $GOPATH
/Users/rjeczalik
```

Reply:
I will revert this and try again.

Reply:
```
$ go test github.com/scylladb/scylla-bench/random
can't load package: package github.com/scylladb/scylla-bench/random: cannot find package "github.com/scylladb/scylla-bench/random" in any of:
        /usr/lib/golang/src/github.com/scylladb/scylla-bench/random (from $GOROOT)
        /home/botond/.go/src/github.com/scylladb/scylla-bench/random (from $GOPATH)
        /home/botond/Scylladb/go/src/github.com/scylladb/scylla-bench/random
$ pwd
/home/botond/ScyllaDB/go/src/github.com/scylladb/scylla-bench
$ echo $GOPATH
/home/botond/.go:/home/botond/Scylladb/go
```

Reply:
Nevermind, I had a typo in my `$GOPATH`. :man_facepalming: